iam-login-service/src/main/java/it/infn/mw/iam/api/requests/exception/UserMismatchException.java

```diff
@@ -0,0 +1,10 @@
+package it.infn.mw.iam.api.requests.exception;
+
+public class UserMismatchException extends IllegalArgumentException {
```

Comment:
This can be removed if we drop the ability to submit group requests for other users